packages/components/src/errorMessage/src/ErrorMessage.tsx

```diff
@@ -0,0 +1,82 @@
+import { forwardRef, type ForwardedRef, type CSSProperties, useContext } from "react";
+import clsx from "clsx";
+import { type StyledComponentProps, useStyledSystem } from "@hopper-ui/styled-system";
+import { ErrorMessageContext } from "./ErrorMessageContext.ts";
+import { useContextProps, type FieldErrorProps as RACFieldErrorProps, FieldErrorContext as RACFieldErrorContext } from "react-aria-components";
+import { WarningIcon } from "@hopper-ui/icons";
+import { cssModule } from "../../utils/src/cssModule.ts";
+import { useRenderProps } from "../../utils/src/useRenderProps.ts";
+import { type TextProps, Text } from "../../Text/index.ts";
+
+import styles from "./ErrorMessage.module.css";
+
+export const GlobalErrorMessageCssSelector = "hop-ErrorMessage";
+
+export interface ErrorMessageProps extends StyledComponentProps<RACFieldErrorProps>, Omit<TextProps, "style" | "className" | "children" | "color" | "content"> {
+    /**
+     * Whether or not to hide the error message icon.
+     * @default true
+     */
+    hideWarningIcon?: boolean;
```

Comment:
Thinking out loud here, should we use a more generic name such as hideDefaultIcon? This would prevent having to rename this if the icon was to change.